README.md

```diff
@@ -1 +1,20 @@
-# Cproject
\ No newline at end of file
+# C project ( C for College)
+
+This is a feedback system project. Through this project, we will improve the outcome of the feedback's given by the
+students for their teachers.
+
+There are many drawbacks of the traditional system . Such as, some of the teachers were able to read that which feedback is given by whom.
+
+So, to improve this we are developing a feedback system  in
+which even the admin will only be able to see the outcome of the feedback was given by students for the respective teachers.
+
+But there is  another problem that some students are not
+regular in colleges(i.e. who come on five days out of thirty)
+gives the feedback due to which it may harm the profile of a good teacher.As no one can judge anyone on the basis of two or three days meet(or can say irregular meet).
+
+So, we will also develop our project in such a way that, the
+feedbacks of such students will not affect the feedback of a teacher to such extent.
+
+   
```

Comment:
Extra empty lines.

Reply:
Ya empty lines because on grammerly it seems to me lack of space .
And i forget to remove them while commiting